core/arch/arm/include/mm/core_mmu.h

```diff
@@ -367,17 +378,20 @@ struct core_mmu_table_info {
 	unsigned level;
 	unsigned shift;
 	unsigned num_entries;
+	struct mmu_context *ctx;
```

Comment:
This is not needed when virtualization isn't enabled. There's a few places where there's a static allocation of this struct, so it will carry a cost.